docs/tools/bench.md

```diff
@@ -33,6 +33,7 @@ The current benchmarks are:
 * `sqrt`: Implements the [Newton–Raphson Method][newton] of approximating the square root of a number to arbitrary precision
 * `sum-sq-diff`: Output the difference between the sum of the squares of the first *n* natural numbers and the square of their sum.
 * `binary-fmt`: Print the binary format for the given positive integer.
+* `primes-between`: Print the primes int the interval `[a, b]`.
```

Comment:
```suggestion
* `primes-between`: Print the primes in the interval `[a, b]`.
```

Looks overall good, but I think still missing the profiling output file? (Hence the failing CI.)

Reply:
@sampsyo I wasn't sure how to run turnt to get the number of instructions for command that takes input. I posted in zulip under #bril about it